### ISSUE TITLE
BUG:  Activate flow schedule on Prefect UI and prefect server

### ISSUE THREAD

**Issue by avlahop** [PrefectHQ/ui#1142]
**Bug Description**

I am trying to activate schedule for a registered flow in a project but I get an error
**Steps To Reproduce**

1. Got to Project Flows
2. From the project in list try to activate schedule via the toggle switch

**Browsers Tested:**
* [x] Chrome
* [ ] Firefox
* [ ] Safari
* [ ] Edge
* [ ] IE

----------
**Notes**

![Screenshot 2021-11-29 at 17 23 46](https://user-images.githubusercontent.com/3893010/143895580-ed30d601-6870-4b6d-8da3-a0d8a05fd1d3.png)

**Comment by zhen0** [comment]
Hi @avlahop - I can only replicate this with a flow that has no schedule. Can you confirm that you have a schedule attached to your flow?  (You can create a schedule in the UI or [in code](https://docs.prefect.io/core/concepts/schedules.html#simple-schedules)).

**Comment by avlahop** [comment]
@zhen0 you are correct. I wasn't aware how the schedule toggle was working. I think you can can proceed with closing this issue. Thank you for you answer!
